platform/feature-api/api/pod-api/src/tests/default.test.ts

```diff
@@ -6,5 +6,6 @@ import { Volume } from "memfs";
 import { getHttpbinUrl } from "@polypoly-eu/fetch-spec";
 
 describe("Mock pod", () => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
in the code below, you've imported FS and then used `as unknown as FS`. Can you use it here too?